### ISSUE TITLE
Implement List and List Item Quick Navigation Keys

### ISSUE THREAD

**Issue by michaelDCurran** [nvaccess/test_nvdaTracImport#146]
**Reported by Iris on 20 Apr 2008 13:40 UTC**
Although NVDA announces lists and their respective list items in the new virtual buffers, the List and list Item shortcut keys fail to perform the desired function.

**Comment by michaelDCurran** [comment]
**Modified by jteh on 20 Apr 2008 22:07 UTC**

**Comment by michaelDCurran** [comment]
**Comment by jteh on 20 Apr 2008 22:38 UTC**
Implemented in r1930.

**Comment by michaelDCurran** [comment]
**Comment by jteh on 20 Apr 2008 22:39 UTC**
Oops.
